dataprofiler/tests/data_readers/test_csv_data.py

```diff
@@ -376,6 +376,24 @@ def test_len_data(self):
                              data.length,
                              msg=input_file['path'])
 
+    def test_is_structured(self):
+        test_dir = os.path.join(test_root_path, 'data')
+        filename = 'csv/blogposts.csv'
+        filename = os.path.join(test_dir, filename)
+
+        # Default construction
+        data = CSVData(filename)
+        self.assertTrue(data.is_structured)
+
+        # With option specifying dataframe as data_format
+        data = CSVData(input_file_path=filename,
+                       options={"data_format": "dataframe"})
+        self.assertTrue(data.is_structured)
+
+        # With option specifying records as data_format
+        data = CSVData(input_file_path=filename,
+                       options={"data_format": "records"})
+        self.assertFalse(data.is_structured)
```

Comment:
Wouldn't we need tests for other stuff?

Reply:
Like what? If its just dependent on data format and I checked the 2 data formats what else can I confirm?

Reply:
I think you need to test all the expected data types: parquet, avro, csv, json

Reply:
I broke out each data type into its own task